src/server/game/Movement/Waypoints/WaypointManager.h

```diff
@@ -68,9 +70,6 @@ class TC_GAME_API WaypointMgr
         }
 
     private:
-        WaypointMgr();
```

Comment:
You should not remove the constructor here - it is intended to be private to enforce singleton pattern